README.adoc

```diff
@@ -363,7 +363,7 @@ Other options include `Debug` (disables optimizations) and `Coverage`.
 |Enables Leak Sanitizer suppressions for libpython3.
 Use it to disable sanitization with older Python version (< 3.9).
 
-|`-DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON`
+|`-DENABLE_LINKTIME_OPTIMIZATION=ON`
 |With CMake 3.9+, compiles the project with LTO (Link Time Optimization) enabled.
```

Comment:
Maybe remove the "With CMake 3.9+" , and "Older versions of CMake", because we now require at least 3.20 in any case

https://github.com/skupperproject/skupper-router/blob/45d75bbfd428ef0ec03239d127125d786f11aa9c/CMakeLists.txt#L20

Reply:
Done !